omero/developers/Matlab.rst

```diff
@@ -267,7 +267,7 @@ argument::
   memory-consuming depending on the amount of data.
 
 To return the orphaned datasets i.e. datasets not in a project, as well as the projects, you can query the second output argument of
-:source:`getProjects <components/tools/OmeroM/src/io/getProjects.m>`::
+:matlab_source:`getProjects <components/tools/OmeroM/src/io/getProjects.m>`::
```

Comment:
Should this not drop `components/tools/OmeroM/`?

Reply:
I too had expected so but no: see https://github.com/ome/omero-matlab/tree/master/components/tools/OmeroM

Reply:
Ah, see https://github.com/ome/omero-matlab/pull/9 cc: @jburel @sbesson How do we want to order all of this?

Reply:
From my side, I certainly approve these changes - see https://github.com/ome/omero-matlab/pull/9#pullrequestreview-160657146 but I understand this has not been tested yet and `omero-matlab` will not be the focus of the immediate next 5.5.0 milestones.

I think we are down to 2 options:
- merging this PR as it is and handling the renaming and the documentation changes as part of the dedicated milestone
- merging https://github.com/ome/omero-matlab/pull/9 immediately and updating the links right now.

If the second option is preferable, I might be able to carry out some minimal testing and report issues directly on the `omero-matlab` repo that we can consider when we come back to it.

Reply:
Thanks, @seb. It being Friday, let's err on the side of merging this PR as opposed merging 9, pushing commits, merging this PR, etc.